pkg/admission/utils.go

```diff
@@ -91,3 +91,12 @@ func (c *customValidator) ValidateDelete(ctx context.Context, obj runtime.Object
 	}
 	return c.validateDelete(ctx, c.Client, obj)
 }
+
+func isStringSliceContains(theStringSlice []string, theString string) bool {
+	for _, s := range theStringSlice {
+		if s == theString {
+			return true
```

Comment:
This could be replaced with https://pkg.go.dev/slices@go1.21.0#Contains

Reply:
of course!